libs/barista-components/filter-field/src/filter-field.scss

```diff
@@ -178,3 +178,13 @@ button.dt-filter-field-clear-all-button-hidden {
 .dt-filter-field-error p {
   margin: 0;
 }
+
+::ng-deep .dt-filter-field-panel .dt-option {
```

Comment:
NIT: please prefix the `::ng-deep` with a `:host` as you break out of the encapsulation of the component.

Reply:
We need to break encapsulation, as this is styled in the cdk overlay portal, hence the `.dt-filter-field-panel` scope